website/src/components/canvas/Sidebar.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Drawer from 'material-ui/Drawer';
+import MenuItem from 'material-ui/MenuItem';
+import FlatButton from 'material-ui/FlatButton';
+
+export default class Sidebar extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {open: false};
+    this.setOpen = this.setOpen.bind(this);
+    this.setClose = this.setClose.bind(this);
+    this.listObject = ['Add New Object',
+                       'Options',
+                       'User List',
+                       'Resize',
+                       'Select Color',
+                       'Sign Out'];
+    this.key;
+    this.listItems = this.listObject.map((item)=><MenuItem key={item.toString()} onTouchTap = {this.setClose}>{item}</MenuItem>);
```

Comment:
@seanhansberry 
listObject / listItems shouldn't be part of the properties of this component since they never change. You can just do it through an array, kind of like [this](http://stackoverflow.com/questions/29149169/how-to-loop-and-render-elements-in-react-js-without-an-array-of-objects-to-map)